documentation/content/tests.rst

```diff
@@ -9,6 +9,12 @@ Kw's tests rely on `shunit2`. The `run_tests.sh` script automatically detects
 can have shunit2 source code in `tests/` (you can clone it from
 https://github.com/kward/shunit2).
 
+In order to run the integration tests, it is necessary to install Podman and  to
+configure it to run in `rootless` mode. Podman  is  available  via  the  default
```

Comment:
I think running as a rootless is already the default for podman. Am I wrong?

Reply:
Yes, but you have to configure rootless mode by adding 65536 subuids (sub-user ids) in order to properly run podman containers. Each one of the subuids will be mapped to a user uid in the containers; that's how podman works under the hood, mapping unprivileged user ids from the host to user ids in the container. If the user running podman does not have subuids, podman may not work properly.